mesonbuild/compilers/mixins/clike.py

```diff
@@ -591,25 +591,26 @@ def cross_sizeof(self, typename: str, prefix: str, env: 'Environment', *,
 
     def sizeof(self, typename: str, prefix: str, env: 'Environment', *,
                extra_args: T.Union[None, T.List[str], T.Callable[[CompileCheckMode], T.List[str]]] = None,
-               dependencies: T.Optional[T.List['Dependency']] = None) -> int:
+               dependencies: T.Optional[T.List['Dependency']] = None) -> T.Tuple[int, bool]:
         if extra_args is None:
             extra_args = []
         if self.is_cross:
-            return self.cross_sizeof(typename, prefix, env, extra_args=extra_args,
-                                     dependencies=dependencies)
+            r = self.cross_sizeof(typename, prefix, env, extra_args=extra_args,
+                                  dependencies=dependencies)
+            return r, False
```

Comment:
I wondered if I instead should make cross_sizeof() return the tuple, but that'd require adjusting cross_compute_int() to look clean, making the change larger.